packages/webpack-cms-plugins/HubSpotAutoUploadPlugin.js

```diff
@@ -47,7 +47,7 @@ class HubSpotAutoUploadPlugin {
           const dest = `${this.dest}/${filename}`;
           upload(this.accountId, asset.existsAt, dest)
             .then(() => {
-              webpackLogger.info(`Uploaded ${dest}`);
+              webpackLogger.info(`Uploaded ${dest} to ${this.accountId}`);
```

Comment:
Can we prefix with "account" so that it is clearer that the number is a portal id?